.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "env": {
+        "node": true,
+        "es6": true
```

Comment:
please remove this line as we do not use es6 syntax/features in our projects at this time